etc/di.xml

```diff
@@ -12,8 +12,38 @@
     <preference for="Reach\Payment\Api\DutyCalculatorInterface" type="Reach\Payment\Model\DutyCalculator" />
     <preference for="Reach\Payment\Api\GuestDutyCalculatorInterface" type="Reach\Payment\Model\GuestDutyCalculator" />
 
+    <preference for="Reach\Payment\Api\NotificationInterface" type="Reach\Payment\Model\ReachNotification" />
+
     <preference for="Reach\Payment\Api\Data\StashResponseInterface" type="Reach\Payment\Model\Api\Data\StashResponse" />
     <preference for="Reach\Payment\Api\Data\DutyResponseInterface" type="Reach\Payment\Model\Api\Data\DutyResponse" />
     <preference for="Reach\Payment\Api\Data\ResponseInterface" type="Reach\Payment\Model\Api\Data\Response" />
     <preference for="Reach\Payment\Api\Data\HttpResponseInterface" type="Reach\Payment\Model\Api\Data\HttpResponse" />
+    <preference for="Reach\Payment\Api\Data\NotificationResponseInterface" type="Reach\Payment\Model\Api\Data\NotificationResponse" />
+
+    <type name="Magento\Framework\Webapi\Rest\Request\DeserializerFactory">
```

Comment:
This is overriding the default Magento `DeserializerFactory`. Needed to add an `item` for all the other types as well (`application/json`, `application/xml`, etc) using the default Magento classes as well, otherwise it wouldn't know how to handle them.